### PR TITLE
Add gardener scheduler integration test to tm testsuite

### PR DESCRIPTION
**Pull request by schrodit**
**What this PR does / why we need it**:
Added the gardener scheduler integration test to tm testsuite

**Special notes for your reviewer**:

**Release note**:

```improvement operator
NONE
```

### CODE REVIEW

File: .ci/testruns/default/templates/testrun-default.yaml [hunk 1]
```diff
@@ -359,8 +362,40 @@ spec:
...
+        value: {{ .Values.shoot.k8sVersion }}
+      - name: SEED
+        type: env
+        value: gcp
+      - name: CLOUDPROFILE
+        type: env
+        value: aws
+      - name: SECRET_BINDING
+        type: env
+        value: core-aws-aws
```
Comment (rfranzke) [inline .ci/testruns/default/templates/testrun-default.yaml:389]: is this correct?

Comment (schrodit) [reply .ci/testruns/default/templates/testrun-default.yaml:389]: @Diaphteiros this is the default naming convention of garden-setup?

Comment (Diaphteiros) [reply .ci/testruns/default/templates/testrun-default.yaml:389]: Yes, the naming scheme for the automatically created provider secrets/secretbindings is `<project>-<provider>-<seed/cloudprofile name>`, see [here](https://github.com/gardener/garden-setup/blob/master/components/gardencontent/coreproject/deployment.yaml#L23).

**Review by rfranzke — APPROVED** [review]
/lgtm

**Review by dguendisch — APPROVED** [review]

**Review by danielfoehrKn — APPROVED** [review]

**Pull request merged**